app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+<title>Writter</title>
```

Comment:
call this file `_head`. the header will be confused with the header of the views